nuxt.config.js

```diff
@@ -64,9 +64,31 @@ export default {
     // https://go.nuxtjs.dev/axios
     '@nuxtjs/axios',
     // https://go.nuxtjs.dev/content
-    '@nuxt/content'
+    '@nuxt/content',
+    'nuxt-i18n'
   ],
 
+  i18n: {
+    lazy: true,
+    langDir: 'locale/',
+    locales: [
+      {
+        code: 'en',
+        file: 'en-US.js',
+        name: 'English'
+      },
+      {
+        code: 'es',
+        file: 'es-ES.js',
+        name: 'Espanol'
+      }
+    ],
+    defaultLocale: 'en',
+    vueI18n: {
+      fallbackLocale: 'en'
```

Comment:
use a environnement variable like `DEFAULT_LANG`
```suggestion
    defaultLocale: process.env.DEFAULT_LANG,
    vueI18n: {
      fallbackLocale: process.env.DEFAULT_LANG
```

In the same idea, you can also replace `'en'` line 11 with `process.env.DEFAULT_LANG`

Reply:
Done, used `DEFAULT_LOCALE` to keep it consistent.